### PR TITLE
Ofir/feedback

### PR DESCRIPTION
**Pull request by OfirZinkerz**
TFL-981
add feedback component +  dist

### CODE REVIEW

File: demo/feedback/assets/svg/completed-v-feedback.svg [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<svg xmlns="http://www.w3.org/2000/svg" x="0px" y="0px"	 viewBox="-1040 834.9 220.4 220.4"  xml:space="preserve">
```
Comment (assafshp) [inline demo/feedback/assets/svg/completed-v-feedback.svg]: ask Oren about what we need here

Comment (OfirZinkerz) [inline demo/feedback/assets/svg/completed-v-feedback.svg]: fixed. 3 icons replaced

**Comment by assafshp** [comment]
Why did you add the file - src/scss/common/_svg-icons.scss ?

File: src/components/feedback/module.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+    angular.module('znk.infra-web-app.feedback',
+        [
+            'ngMaterial',
+            'znk.infra.popUp',
+            'pascalprecht.translate',
+            'znk.infra.auth',
+            'znk.infra.analytics',
+            'znk.infra.general',
+            'znk.infra.user',
```
Comment (assafshp) [inline src/components/feedback/module.js]: are you using it ?

File: src/components/feedback/controllers/feedback.controller.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+                if (self.feedbackForm.$valid) {
+                    self.startLoader = true;
+                    var authData = AuthService.getAuth();
+                    var postData = angular.copy(self.feedbackData);
+
+                    postData.tags = ENV.firebaseAppScopeName;
+                    postData.message += (ENTER_KEY + ENTER_KEY);
+                    postData.message += ' APP-NAME: ' + ENV.firebaseAppScopeName + ', UID: ' + (authData ? authData.uid : 'N/A');
+
+                    $http.post(ENV.doorBellSubmitURL, (postData)).then(function (data) {
```
Comment (assafshp) [inline src/components/feedback/controllers/feedback.controller.js]: Is it ok to do the post here ? 
we usually use $http in service or factory

File: src/components/feedback/module.js (hunk 2 above)
Comment (OfirZinkerz) [inline src/components/feedback/module.js]: UserProfileService use it

**Pull request closed without merging**